azure_arc_servers_jumpstart/docs/arc_policies_mma.md

```diff
@@ -0,0 +1,109 @@
+# Overview
+
+The following README will guide you on how to use Arc for servers to assign Azure Policies to VMs outside of Azure, wether they are on-premise or other clouds. With this feature you can now use Azure Policies to audit settings in the operating system of an Azure Arc connected servers, if a setting is not compliant you can also trigger a remediation task. 
+
+In this case we will assign a policy to audit if the Azure Arc connected machine has the (Microsoft Monitoring Agent) MMA agent installed, if not, we will use the extensions feature to automatically deploy it to the VM, an enrollment experience that levels to Azure VMs. This approach can be used to make sure all your servers are onboarded to services such as: Azure Monitor, Azure Security Center, Azure Sentinel, etc. 
```

Comment:
slightly awkward wording. Suggested edit:
We will assign a [built-in policy](https://docs.microsoft.com/en-us/azure/governance/policy/samples/built-in-policies) to audit if an Azure Arc connected machine has the Microsoft Monitoring Agent (MMA) installed. If it does not, the policy will automatically deploy the agent to the VM.